src/webextension/locales/en-US/manage.ftl

```diff
@@ -1,5 +1,11 @@
-add-item = Add Item
```

Comment:
@jimporter Would you like me to go through and inventory the strings and change them to what @changecourse has proposed?

If so: I can do it here or in my own branch to merge into this one. Your call. 😁

Reply:
With your hard work on getting everything localized, it turns out it's really easy to update strings. I went through the invision docs and here's a branch in case you want to use it: https://github.com/mozilla-lockbox/lockbox-extension/commits/update-ux-strings

(we can do a PR into this PR if we want to get all complicated about it)